main.py

```diff
@@ -0,0 +1,53 @@
+import random  # import random module
+
+# create variables to store names and number of names
+name_list = []
+name_list_items = 0
+number = 0
+selected_name = []
+
+def insert_names():
+    '''This function will insert names into the name_list'''
+    global name_list
+    global name_list_items
+    while True:
+        input_name = input('Enter random names... to STOP press \'c\' : ')
+        name_list.append(input_name)
+        if input_name == 'c':
+            name_list.remove('c')

```

Comment:
Check the input_name before adding anything to the list. This will save you from removing 'c' immediately after adding it which is more logical.